src/rviz/ogre_helpers/movable_text.cpp

```diff
@@ -41,16 +41,16 @@
 
 #include "movable_text.h"
 
-#include <OGRE/OgreVector3.h>
-#include <OGRE/OgreQuaternion.h>
-#include <OGRE/OgreRoot.h>
-#include <OGRE/OgreCamera.h>
-#include <OGRE/OgreSceneNode.h>
-#include <OGRE/OgreMaterialManager.h>
-#include <OGRE/OgreHardwareBufferManager.h>
-#include <OGRE/Overlay/OgreFontManager.h>
-#include <OGRE/Overlay/OgreFont.h>
-#include <OGRE/OgreUTFString.h>
+#include <OgreVector3.h>
+#include <OgreQuaternion.h>
+#include <OgreRoot.h>
+#include <OgreCamera.h>
+#include <OgreSceneNode.h>
+#include <OgreMaterialManager.h>
+#include <OgreHardwareBufferManager.h>
+#include <Overlay/OgreFontManager.h>
+#include <Overlay/OgreFont.h>
+#include <OgreUTFString.h>
```

Comment:
this makes https://github.com/ros-visualization/rviz/commit/a4919f52fb37da074fcea1648e257691037a3feb superfluous as proposed originally in #1645  :wink: